docs/src/publications.md

```diff
@@ -0,0 +1,11 @@
+# Papers and preprints using Oceananigans.jl
+
+If you have work using Oceananigans.jl that you would like to have listed here, please open a pull request to add it or let us know!
+
+1. Wagner, G. L., Chini, G. P., Ramadhan, A., Gallet, B., & Ferrari, R. (2021). [Near-inertial waves and turbulence driven by the growth of swell](https://doi.org/10.1175/JPO-D-20-0178.1), Journal of Physical Oceanography, in press. DOI: [10.1175/JPO-D-20-0178.1](https://doi.org/10.1175/JPO-D-20-0178.1)
+
+1. Buffett, B. A. (2021). [Conditions for turbulent Ekman layers in precessionally driven flow](https://doi.org/10.1093/gji/ggab088), Geophysical Journal International, 226, 1, pp. 56–65. DOI: [10.1093/gji/ggab088](https://doi.org/10.1093/gji/ggab088)
+
+1. Bhamidipati, N., Souza, A.N. & Flierl, G.R., (2020). [Turbulent mixing of a passive scalar in the ocean mixed layer](https://doi.org/10.1016/j.ocemod.2020.101615). Ocean Modelling, 149, 101615. DOI: [10.1016/j.ocemod.2020.101615](https://doi.org/10.1016/j.ocemod.2020.101615)
+
+1. Souza, A. N., Wagner, G. L., Ramadhan, A., Allen, B., Churavy, V., Schloss, J., Campin, J. M., Hill, C., Edelman, A., Marshall, J., Flierl, G., & Ferrari, R. (2020). [Uncertainty quantification of ocean parameterizations: Application to the K‐Profile‐Parameterization for penetrative convection](https://doi.org/10.1029/2020MS002108). Journal of Advances in Modeling Earth Systems, 12, e2020MS002108. DOI: [10.1029/2020MS002108](https://doi.org/10.1029/2020MS002108)
```

Comment:
Seems all listed papers have number 1 in front of them?

Reply:
Yup! Markdown actually ignores your numbers and numbers them automatically: https://www.markdownguide.org/basic-syntax/#ordered-lists

So I'm using `1. ` in front of each entry in case we want to re-order them in the future and don't want to adjust all the numbers.